app-editors/retext/retext-7.2.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# Please don't add pypy support before testing if it's actually supported. The
+# old compat matrix is no longer accessible as of 2021-02-13 but stated back
+# in 2020-07-05 that PyQt5 was explicitly not supported.
+PYTHON_COMPAT=( python3_{8,9} )
+
+inherit distutils-r1 optfeature virtualx xdg-utils
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
This needs to be set before inheriting *distutils-r1.eclass*. But it looks like false anyway.

Reply:
Right, needs to be bdepend - I fixed it locally.